Makefile

```diff
@@ -6,8 +6,10 @@ export GO15VENDOREXPERIMENT=1
 include versioning.mk
 
 DEV_ENV_IMAGE := quay.io/deis/go-dev:0.9.0
+SWAGGER_IMAGE := quay.io/goswagger/swagger:0.5.0
```

Comment:
nevermind, you pointed me to if offline: https://github.com/go-swagger/go-swagger/blob/master/Dockerfile